### PR TITLE
fixed minor typos

### PR DESCRIPTION
**Pull request by hamidzr**

### CODE REVIEW

**Comment by jonathanslenders** [comment]
``initialisation`` is valid English. Just not American English. ;)(
See: https://en.wiktionary.org/wiki/initialization

Two questions:
- Could you rephrase the commit message to something like "Use American English for consistency."?
- Further, if you see more spelling mistakes, I appreciate if it's just one commit that contains all fixes, rather then a one character pull request.

**Comment by hamidzr** [comment]
My bad there is nothing wrong with how it was spelled in the first place. Feel free to close the PR. 
And yes squashing the commits sounds good
